src/in_range.rs

```diff
@@ -43,6 +43,10 @@ where
         unsafe { assert_unchecked(self.in_bounds()) };
         self.0
     }
+
+    pub const fn const_get(&'static self) -> T {
+        self.0
+    }
```

Comment:
```suggestion
    /// This doesn't [`assert_unchecked`] that the value is in bounds (for optimization)
    /// because [`Self::in_bounds`] is not a `const fn` due to its usage of `trait`s.
    /// However, because this is meant to be called in a `const` context,
    /// everything should be known already, and the [`assert_unchecked`]
    /// should be unnecessary for optimization.
    pub const fn const_get(self) -> T {
        self.0
    }
```